src/csv_to_confluence.py

```diff
@@ -36,6 +36,7 @@
 
 from common import Main, CSVHandler, ConfluenceConnection, ConfluenceNodeMapper, ErrorCSVHandler, InfoCSVHandler, ResourceLoader, SingletonABCMeta, \
     SingletonMeta, TimestampHandler
+from src.file_backup_service import FileBackupManager
```

Comment:
This line has no use

Reply:
Class FileBackupManager is used in class ConfluencePageHandlerManager to upload a backup to confluence.

Reply:
FileBackupManager should be decoupled from csv_to_confluence.py and executed seperately